src/en/space-station-14/mapping/guides/general-guide.md

```diff
@@ -12,24 +12,26 @@ This guide is separated into **workflow** (setup, starting a map, saving, useful
 If you've read "Workflow" and are looking for a refresher:
 
 1. To create a new map: `mapping ID`, or
-   To edit an existing map: `mapping ID PATH` (e.g. `mapping 3 Maps/saltern.yml`)
-   Note that `ID` must be the map number of a map that does not already exist. Use `lsmaps` to see a list of maps that are currently in use.
+   To edit an existing map: `mapping ID PATH` (e.g. `mapping 1000 Maps/saltern.yml`)
+   Note that `ID` must be the map number of a map that does not already exist. Use `lsmaps` to see a list of maps that are currently in use. A higher ID is recommended to avoid colliding with existing ones.
    
-2. If desired, `loadmapacts`
+2. If desired, `mappingclientsidesetup` to use the default mapping setup.
 
-3. Make changes
+3. Make all your changes.
    
-4. If needed, run `fixgridatmos GID`
+4. If you have created new rooms, run `fixgridatmos GID` before saving.
```

Comment:
step 4 occurs before 5. "before saving" is implied, so only add this if it causes project loss if forgotten.

Same feedback for "you".